### PR TITLE
Feature/shravan1k/977193165 refactor right line body

### PR DESCRIPTION
**Pull request by shravan1k**
### Administrative Info
Monday Board ID: 940408300
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Refactored body that is aligned right into tables

### Testing
How did you confirm your changes worked? 
- Viewed them on gmail desktop, gmail mobile, and apple mail

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Gmail Desktop | Gmail Mobile | Apple Mail |
| --- | --- | --- |
| Screen Shot 2021-01-17 at 10 15 18 PM | IMG_0609 | IMG_0610 |

### CODE REVIEW

File: html/rightlinebody.html [hunk 1]
```diff
@@ -15,8 +15,149 @@
...
+                                                            <td class="sk-td sk-img-right">
+                                                                <img class="sk-img" src ="https://drive.google.com/uc?export=view&id=18JqJmYshovA9u3e0HCikV9NVD9wpDVWJ">
+                                                            </td>
+                                                        </tr>
+                                                    </table>
+                                                </td>
+                                            </tr>
+                                        </table>
+                                    </td>
+                                    <td class="sk-td" style="padding-right: 30px; padding-top: 85px;">
```
Comment (wes-chen) [inline html/rightlinebody.html:149]: are we refactoring all of the inline styles out?

File: newsletter.css [hunk 2]
```diff
@@ -73,58 +107,46 @@
...
+    padding-left: 110px;
+    padding-top: 8px;
+}
+
 .sk-detail-pad{
     padding-top: 8px;
 }
 
+.sk-right{
+    text-align: -webkit-right;
```
Comment (wes-chen) [inline newsletter.css:145]: does webkit only work for apple mail? or for gmail mobile / gmail desktop as well

**Review by wes-chen — CHANGES_REQUESTED** [review]

**Review by eliasfang — APPROVED** [review]
After addressing Wesley's comments, the table formatting and everything look right to me. I think sometimes you might need inline styling for those specific elements (unless you add another class name). And good catch with changing the css file path!

File: newsletter.css (hunk 2 above)
Comment (shravan1k) [reply newsletter.css:145]: It works for all of them. Without webkit, nothing was aligning for apple mail.

File: html/rightlinebody.html (hunk 1 above)
Comment (shravan1k) [reply html/rightlinebody.html:149]: yeah that will probably be better for the generating the template, will make the changes

**Review by wes-chen — APPROVED** [review]

**Pull request merged**